### PR TITLE
3.6 Testing Fixes 1

### PR DESCRIPTION
**Pull request by Tiny-DM**
## What type of PR is this.
1. [X] Bug
2. [X] Change
3. [X] Enhancement

### What have you changed and why?
Information: 
Added faction information to the SFP and IFA factions
Corrected a misspelling in "arctic"
Corrected typos in the hint if the new battle menu is disabled
Artillery markers will now delete globally when the interface is exited
Garage hints should execute onto the actual player now
The garage should now also remove the vehicle from the pool server side as well, fixing a duping exploit
Added additional logging to the sale of vehicles.
Slightly restructured the garage UI to fix an edge case if ACE was loaded
Added SFP hardcoding to the factions tab data
The factions tab addons will now fall back on the internal name of the dependency if none is found
    
### Please verify the following and ensure all checks are completed.

1. [X] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [X] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Need to test if the garage sell works on DS now

### CODE REVIEW

File: A3A/addons/core/Stringtable.xml [hunk 1]
```diff
@@ -16437,6 +16437,33 @@
...
+      </Key>
+      <Key ID="STR_A3A_templates_lore_SFP_SAF15">
+        <Original>The Swedish Defense Force was never focused on any exploratory ventures into surrounding nations, instead opting to maintain its alliances and fortify the Swedish border from any kind of enemy attack. They use a variety of homegrown weapons, often using the AK5, a variant of the FN FNC.</Original>
+      </Key>
+      <Key ID="STR_A3A_templates_lore_IFA_AI_SOV">
+        <Original>The Soviets during World War II worked with the Allies after Russian territory was invaded during the Wehrmacht's destructive path to power. They most often use the PPSH-41 submachine gun and Mosin-Nagant rifle, with few exceptions.</Original>
+      </Key>
+      <Key ID="STR_A3A_templates_lore_IFA_AI_ALLIES">
+        <Original>The Allies of World War II were a coalition of nations dedicated to fighting the Wehrmacht's ruthless invasion of it's neighboring countries. They most often use American-exported weapons, such as the M1 Garand, M1A1 Thompson, and M1 Carbine.</Original>
```
Comment (jaj22) [inline A3A/addons/core/Stringtable.xml:16450]: it's -> its neighboring countries.

File: A3A/addons/core/Stringtable.xml [hunk 2]
```diff
@@ -16437,6 +16437,33 @@
...
+      </Key>
+      <Key ID="STR_A3A_templates_lore_IFA_AI_UK">
+        <Original>The United Kingdom was at the forefront of World War II, a final show of force from an once-great empire. They commonly use a variety of home-grown weapons like the Sten and Enfield, mixed with a variety of American export weaponry.</Original>
```
Comment (jaj22) [inline A3A/addons/core/Stringtable.xml:16453]: an -> a one-great empire.

File: A3A/addons/core/Stringtable.xml [hunk 3]
```diff
@@ -16437,6 +16437,33 @@
...
+      </Key>
+      <Key ID="STR_A3A_templates_IFA_Reb_FFI">
+        <Original>The French Resistance is battling the Wehrmacht invasion of their country, along with the Allied powers who they believe are being far too heavy handed with civilian casualities and need to be stopped. They use a much older but wealthy supply of Walther PPKs.</Original>
```
Comment (jaj22) [inline A3A/addons/core/Stringtable.xml:16456]: wealthy -> healthy supply? Never seen that anyway.

**Review by jaj22 — APPROVED** [review]
Minor grammar corrections, fix if you care. Rest of it is fine.

File: A3A/addons/core/Stringtable.xml (hunk 1 above)
Comment (ante185) [inline A3A/addons/core/Stringtable.xml:16450]: in this particular instance the militia of the template represent the UK forces and aren't phased out as completely as in other conventional templates.
Use that information as you see fit

File: A3A/addons/core/Stringtable.xml (hunk 2 above)
Comment (rautamiekka) [reply A3A/addons/core/Stringtable.xml:16453]:
```suggestion
        <Original>The United Kingdom was at the forefront of World War II, a final show of force from a once-great empire. They commonly use a variety of home-grown weapons like the Sten and Enfield, mixed with a variety of American export weaponry.</Original>
```

File: A3A/addons/core/Stringtable.xml (hunk 1 above)
Comment (rautamiekka) [reply A3A/addons/core/Stringtable.xml:16450]:
```suggestion
        <Original>The Allies of World War II were a coalition of nations dedicated to fighting the Wehrmacht's ruthless invasion of its neighboring countries. They most often use American-exported weapons, such as the M1 Garand, M1A1 Thompson, and M1 Carbine.</Original>
```

File: A3A/addons/core/Stringtable.xml (hunk 3 above)
Comment (rautamiekka) [reply A3A/addons/core/Stringtable.xml:16456]:
```suggestion
        <Original>The French Resistance is battling the Wehrmacht invasion of their country, along with the Allied powers who they believe are being far too heavy-handed with civilian casualities and need to be stopped. They use a much older but healthy supply of Walther PPKs.</Original>
```

**Pull request merged**